src/goto-instrument/dump_c_class.h

```diff
@@ -49,6 +52,7 @@ class dump_ct
   symbol_tablet copied_symbol_table;
   const namespacet ns;
   languaget *language;
+  const bool use_all_headers;
```

Comment:
This variable doesn't appear to be used - I deleted it and constructed system_symbols with it directly.